gremlin-go/driver/graphTraversal.go

```diff
@@ -670,3 +675,99 @@ func (g *GraphTraversal) Write(args ...interface{}) *GraphTraversal {
 	g.bytecode.addStep("write", args...)
 	return g
 }
+
+type transaction struct {
+	g                      *GraphTraversalSource
+	sessionBasedConnection *DriverRemoteConnection
+	remoteConnection       *DriverRemoteConnection
+	isOpen                 bool
```

Comment:
If possible, can we use the inverse `isClosed` since it would align with the rest of our types.

Reply:
same https://github.com/lyndonbauto/tinkerpop/blob/gremlin-go-ms5/gremlin-go/driver/resultSet.go#L53